hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -156,15 +155,15 @@ private int doCluster(JavaSparkContext jsc) throws Exception {
   }
 
   @TestOnly
-  public Option<String> doSchedule() throws Exception {
```

Comment:
return the schedule instant time will be more clear.

Reply:
The schedule instant time is already in `HoodieClusteringJob.Config`.  If `doSchedule()` succeeds and returns 0, we should be able to get the clustering instant time from the config.
I am trying to use the same patern as `doSchedule()` in `HoodieCompactor`.  Correct me if I misunderstand it.